airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "orchestrator"
-version = "0.4.0"
+version = "0.4.1"
```

```suggestion
version = "0.5.0"
```
Since we added a new job